compiler/src/dotty/tools/dotc/transform/TupleOptimizations.scala

```diff
@@ -24,7 +24,8 @@ class TupleOptimizations extends MiniPhase with IdentityDenotTransformer {
   def phaseName: String = "genericTuples"
 
   override def transformApply(tree: tpd.Apply)(implicit ctx: Context): tpd.Tree = {
-    if (tree.symbol == defn.DynamicTuple_dynamicCons) transformTupleCons(tree)
+    if (!tree.symbol.exists || tree.symbol.owner != defn.DynamicTupleModuleClass) super.transformApply(tree)
```

Comment:
You don't need a `super.transformApply` in a MiniPhase. Just return `tree` itself.

Reply:
modified